worker/src/app.py

```diff
@@ -66,7 +67,10 @@ async def lifespan(context: ContextRepo):
 def document_loader(s3_client: S3Client, env: Settings):
     @chain
     def wrapped(file: File):
-        return UnstructuredDocumentLoader(file, s3_client, env).lazy_load()
+        file_raw = BytesIO()
+        s3_client.download_fileobj(Bucket=file.bucket, Key=file.key, Fileobj=file_raw)
```

Comment:
can we pass the 'Body' field of get_object to the documentloader to avoid this download? It's also a byte stream so that should work?

Reply:
ah ok yeah in which case let's pull it. We should think about cleaning these files up at some point but maybe don't need to worry in this release. Restaring/scaling the workers will clear out all files anyway so should be ok for a bit